addendum.dict

```diff
@@ -339,13 +351,18 @@ Schumer	𐑖𐑵𐑥𐑼
 Schwarzenegger	𐑖𐑢𐑹𐑑𐑕𐑩𐑯𐑧𐑜𐑼
 Secheron	𐑕𐑧𐑖𐑼𐑪𐑯
 Servox	𐑕𐑺𐑝𐑪𐑟
+Sega 𐑕𐑧𐑜𐑩
 Shemekia	𐑖𐑩𐑥𐑰𐑒𐑾
 Shoolbred	𐑖𐑫𐑤𐑚𐑮𐑧𐑛
 Simms	𐑕𐑦𐑥𐑟
 Sinead	𐑖𐑦𐑯𐑱𐑛
+Sony 𐑕𐑴𐑯𐑰
```

Comment:
I'd prefer this to be 𐑕𐑴𐑯𐑦 to comply with principle 11 at https://www.shavian.info/spelling/.